perception/autoware_multi_object_tracker/config/input_channels.param.yaml

```
@@ -71,6 +71,17 @@
         optional:
           name: "camera_lidar_fusion"
           short_name: "CLf"
+      # CAMERA-LIDAR-IRREGULAR
+      camera_lidar_fusion_irregular:
+        flags:
+          can_spawn_new_tracker: true
+          can_trust_existence_probability: true
```

```suggestion
          can_trust_existence_probability: false
```
if the detector do not have proper confidence value, it is better to set false